build.proj

```diff
@@ -214,109 +174,70 @@
-    <!-- <CallTarget targets="ChangeLogCheck" ContinueOnError="false" /> -->
-
-    <MakeDir  Directories="$(PackageDirectory)"
-              Condition="'$(Scope)' != 'Stack'" />
-    <MakeDir  Directories="$(StackPackageFolder)"
-              Condition="'$(Scope)' == 'All' or '$(Scope)' == 'Stack'" />
-
-    <MSBuild  Projects="@(CmdletSolutionsToBuild)"
-              Targets="Build"
-              Properties="Configuration=$(Configuration);Platform=Any CPU"
-              BuildInParallel="$(BuildInParallel)"
-              ContinueOnError="false" />
-
-    <MSBuild  Projects="@(StackSolution)"
-              Targets="Build"
-              Properties="Configuration=$(Configuration);Platform=Any CPU"
-              BuildInParallel="$(BuildInParallel)"
-              Condition="'$(Scope)' == 'All' or '$(Scope)' == 'Stack'"
-              ContinueOnError="false" />
-
-    <Exec Command="$(PowerShellCommandPrefix) &quot;$ProgressPreference = 'SilentlyContinue';. $(LibraryToolsFolder)\GenerateHelp.ps1 -ValidateMarkdownHelp -GenerateMamlHelp -BuildConfig $(Configuration) -FilteredModules '@(ModulesChanged)' &quot;"
-          Condition="'$(SkipHelp)' == 'false'"
-          ContinueOnError="false"/>
-
-    <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\UpdateModules.ps1 -BuildConfig $(Configuration) -Scope $(Scope) &quot;"
-          Condition="'$(CodeSign)' == 'false'"/>
-
-    <CallTarget Targets="CodeSignBinaries" Condition="'$(CodeSign)' == 'true'" />
-
-    <Exec ContinueOnError="false"
-      Command="&quot;$(PowerShellCommand)&quot; -NonInteractive -NoLogo -NoProfile -Command &quot; . $(LibraryToolsFolder)\NewOutputTypeIndex.ps1 -OutputFile $(PackageDirectory)\outputtypes.json -BuildConfig $(Configuration) &quot;"
-      Condition="('$(Scope)' == 'All' or '$(Scope)' == 'Latest') and $(CodeSign) == 'true'" />
-
-    <Message Importance="high" Text="Running Static Analyser" />
-    <CallTarget targets="DependencyAnalysis" ContinueOnError="ErrorAndContinue" />
-    <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\CheckAssemblies.ps1 -BuildConfig $(Configuration) &quot;" />
-    <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\CleanupBuild.ps1 -BuildConfig $(Configuration) &quot;" />
-
-    <Exec ContinueOnError="false"
-          Command="&quot;$(PowerShellCommand)&quot; -NonInteractive -NoLogo -NoProfile -Command &quot; . $(LibraryToolsFolder)\NewHelpIndex.ps1 -OutputFile $(PackageDirectory)\index.json -BuildConfig $(Configuration) &quot;"
-          Condition="('$(Scope)' == 'All' or '$(Scope)' == 'Latest') and $(CodeSign) == 'true'" />
-  </Target>
-
-  <!--
-    Build .Net Core Cmdlets
-    Fully self contained
-  -->
-  <Target Name="BuildNetCore">
+  <!-- TODO: Reimplement the FilterBuild and update RepoTasks to NetStandard -->
+  <!-- <Target Name="Build" DependsOnTargets="RestoreNugetPackages;BuildMsBuildTask;FilterBuild"> -->
+  <Target Name="Build">
```

Comment:
This should have the dependencies above, I believe (RestoreNugetPackages; BuildMsBuildTask)

Reply:
Those do not apply at all and aren't removed only because the `RepoTasks` project is not converted to NetStandard yet. `FilterBuild` should be added once it is re-implemented. The other 2 targets are non-applicable.